blocks/CloudImageEditor/src/state.js

```diff
@@ -1,5 +1,6 @@
+import { createCdnUrl, createCdnUrlModifiers } from '../../../utils/cdn-utils.js';
+import { transformationsToString } from './lib/cdnUtils.js';
```

Comment:
is it possible to leave only one cdn-utils entry?

Reply:
Not really. I renamed it to `transformationsUtils` to avoid naming collision. 
It contain higher level api to serialize transformations object to cdn modifiers string, with operation ordering.
I wish we can move this to cdn-utils in near future but for now it's not usable out of cloud editor scope.